OpenSim/Simulation/Model/Marker.h

```diff
@@ -45,7 +45,13 @@ class OSIMSIMULATION_API Marker : public Station {
     OpenSim_DECLARE_CONCRETE_OBJECT(Marker, Station);
 
 class Body;
-
+public:
+    //==============================================================================
+    // PROPERTIES
+    //==============================================================================
+    OpenSim_DECLARE_PROPERTY(fixed, bool,
+        "Flag(true or false) specifying whether or not a marker "
+        "should be kept fixed in the marker placement step.  i.e. If false, the marker is allowed to move.");
```

Comment:
would prefer "Flag(true or false) that specifies whether or not the marker is fixed during marker placement (a step of the ScaleTool). If false, the MarkerPlacer is free to move the model marker's location to match its experimental counterpart." 
Or something like that.